Adding New fields + fields from Kong OSS PR  8841 (#3965)
Kong/docs.konghq.com: app/_hub/kong-inc/request-termination/2.0.x.md

    - name: trigger
      required: false
      value_in_examples: "X-Ref-ID-Key"
      datatype: string
      description: The header or query parameter key to trigger plugin execution.
    - name: trigger_value
      required: false
      value_in_examples: "X-Ref-ID-Value"
      datatype: string
      description: The trigger conf. value to trigger plugin execution.
    - name: avert
      required: false
      value_in_examples: "X-Ref-ID-Key"
      datatype: string
      description: The header or query parameter key to avert plugin execution.
    - name: avert_value
      required: false
      value_in_examples: "X-Ref-ID-Value"
      datatype: string
      description: The avert conf. value to avert plugin execution.
    - name: echo
      required: false
      value_in_examples: true
      datatype: boolean
      description: When enabled, plugin will echo the client request back as the http response.
  extra: |
    Once applied, every request (within the configured plugin scope of a Service,
    Route, Consumer, or global) will be immediately terminated by
    sending the configured response.

---

## New in 2.2.0

> Addition of the avert option, which prevents plugin execution when matched. Opposite from the trigger option

Reviewer: ```suggestion
Added the `avert` option, this prevents plugin execution when matched. `avert` is like the opposite of the `trigger` option 
```
